cumulusci/cumulusci.yml

```diff
@@ -895,6 +895,24 @@ flows:
             2:
                 task: uninstall_managed
 
+    install_2gp_commit:
+        group: Install / Uninstall
+        description: Install the 2GP package for the current commit
```

Comment:
We could expand this description to explain that it's expecting a commit status with the version_id to be set for the "Build Feature Test Package" context.